components/alerts/style.css

```diff
@@ -0,0 +1,53 @@
+.alert__content {
+  border: 1px solid var(--gray-light2);
```

Comment:
You can name it like --grey-light-primary and grey-light-secondary instead of digits

Reply:
sure thanks for pointing out. will update it